tests/components/zwave_js/test_update.py

```diff
@@ -252,14 +251,57 @@ async def test_update_entity_dead(
 
     event = Event(
         "alive",
-        data={"source": "node", "event": "alive", "nodeId": multisensor_6.node_id},
+        data={"source": "node", "event": "alive", "nodeId": zen_31.node_id},
     )
-    multisensor_6.receive_event(event)
+    zen_31.receive_event(event)
     await hass.async_block_till_done()
 
     # Now that the node is up we can check for updates
     assert len(client.async_send_command.call_args_list) > 0
 
     args = client.async_send_command.call_args_list[0][0][0]
     assert args["command"] == "controller.get_available_firmware_updates"
-    assert args["nodeId"] == multisensor_6.node_id
+    assert args["nodeId"] == zen_31.node_id
+
+
+async def test_update_entity_ha_not_running(
```

Comment:
This test is using a socket since the web server is started. Maybe since we're calling `hass.async_start`?